website/markdown/docs/commands/doc.mdx

```diff
@@ -0,0 +1,45 @@
+---
+name: Generate documentation
+excerpt: 'Learn how to generate documentation for your Xcode projects.'
+---
+
+# Generate documentation
+
+Leveraging on [swift-doc](https://github.com/SwiftDocOrg/swift-doc), we facilitate a convenient way to generate documentation for your Xcode projects. It is as simple as running:
```

Comment:
I'd start this section with a bit of context on what motivated us to build this functionality:

> When working in modular codebases, it might be useful to check out the documentation of the public API of a given module. For that reason, Tuist provides a command, `tuist doc`, that given a target, it generates its documentation and opens it on the browser. Under the hood, it uses [swift-doc](https://github.com/SwiftDocOrg/swift-doc).